querybook/webapp/resource/queryReview.ts

```diff
@@ -2,9 +2,15 @@ import { IQueryReview } from 'const/queryExecution';
 import ds from 'lib/datasource';
 
 export const QueryReviewResource = {
-    getReviewsCreatedByMe: () =>
-        ds.fetch<IQueryReview[]>('/query_review/created_by_me/'),
+    getReviewsCreatedByMe: (limit: number, offset: number) =>
+        ds.fetch<IQueryReview[]>('/query_review/created_by_me/', {
+            limit,
+            offset,
+        }),
 
-    getReviewsAssignedToMe: () =>
-        ds.fetch<IQueryReview[]>('/query_review/assigned_to_me/'),
+    getReviewsAssignedToMe: (limit: number, offset: number) =>
+        ds.fetch<IQueryReview[]>('/query_review/assigned_to_me/', {
+            limit,
+            offset,
+        }),
```

Comment:
we have 2 routes that does the same thing? let's use a generic route `/query_review/<review_type>/`